src/legacy/core_plugins/visualizations/public/index.scss

```
@@ -1,3 +1,4 @@
 @import 'src/legacy/ui/public/styles/styling_constants';
 
-@import './embeddable/_index';
+@import './embeddable';
+@import './np_ready/public';
```

Sorry, I think there was some confusion, but you still need to explicitly write out the `index` portion of the import. SASS doesn't automatically look for an index file. 
```suggestion
@import './np_ready/public/index';
```

---

Thank you. was confused by still see everything working locally. Looks like cache.